extension/wrapper.cpp

```diff
@@ -237,7 +340,7 @@ Satrec_len(PyObject *self) {
 }
 
 static PySequenceMethods SatrecArray_as_sequence = {
-    sq_length : Satrec_len,
+    .sq_length = Satrec_len,
```

Comment:
Once the PR is nearly ready, I'll plan on trying it out on the `release-test` branch to see what Mac OS X and Windows think of this syntax; I can't remember if they both like them or not.